.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,63 @@
+# Sample workflow for building and deploying a Hugo site to GitHub Pages
+name: Deploy site to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+# Default to bash
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  # Build job
+  build:
+    # Specify runner +  build & upload the static files as an artifact
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build static files
+        id: build
+        run: |
+          files=$(find . -iname \*.yaml)
+            for file in $files; do
+                echo "Processing $file"
+                mkdir -p build_outputs_folder/$(dirname $file)
+                cp $file build_outputs_folder/$file
+            done
```

Comment:
This script seems to be a bit complicated (but also an ok template).
We could just upload everything and be fine...